src/ktopt_planning_context.cpp

```diff
@@ -248,29 +253,83 @@ void KTOptPlanningContext::transcribePlanningScene(const planning_scene::Plannin
     }
     if (object == OCTOMAP_NS)
     {
-      RCLCPP_ERROR(getLogger(), "skipping octomap for now ...");
+      RCLCPP_INFO(getLogger(), "skipping octomap for now ...");
```

Comment:
Make this a warning and say explicitly: Octomap parsing not supported at the moment